pino.js

```diff
@@ -30,8 +32,15 @@ function pino (opts, stream) {
     istream = iopts
     iopts = defaultOptions
   }
-  istream = istream || process.stdout
   iopts = Object.assign({}, defaultOptions, iopts)
+  if (iopts.extreme && iopts.prettyPrint) throw Error('cannot enable pretty print in extreme mode')
+  istream = istream || process.stdout
+  if (iopts.prettyPrint) {
+    var pstream = pretty(iopts.prettyPrint)
+    var origStream = istream
+    pstream.pipe(origStream)
```

Comment:
we can because the logger will be there if an error is ever emitted. It's never emitted synchronously during `pipe` with streams.